src/docs/content/observability/observability-scenario/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Observability Scenario"
+date: 2020-10-26 T16:04:15-06:00
+draft: false
+showChildren: true
+---
+
+## Introduction
+
+In this section we will try to understand Observability on Oracle Cloud Infrastructure (OCI) by simulating a failure scenario on MuShop application.
+
+![Failure-scenario](../images/observability-scenario.png)
+
+## Scenario Details
+- User logs in to MuShop and Orders items with value more than $105.
+- User notices the Order processing failure.
+- User gets notified on the Payment failures. 
+  In a production setup, application operations team would get paged.
+- User analyzes the failure using Oracle Cloud Infrastructure(OCI) Observability Services.
```

Comment:
Using the observability service to monitor for a functional scenario like this seems a bit unnatural. For instance, perhaps the Monitoring tool can detect say N failed attempts within P seconds and trigger a fraud alert that pages ops to disable the user ?

Reply:
The idea for initial release was to just show that domain specific business metrics are possible using a simple payment failure situation and alerting plus skimming through the logs to find out what happened. We can keep building on top of it as we go. fyi: with monitoring we can only send out time based data and we do not send any user id's there.